homeassistant/components/matter/light.py

```diff
@@ -220,7 +220,7 @@ def _get_brightness(self) -> int:
         return round(
             renormalize(
                 level_control.currentLevel,
-                (level_control.minLevel, level_control.maxLevel),
+                (level_control.minLevel or 0, level_control.maxLevel or 254),
```

Comment:
Is 0 an allowed max level in the spec? The code currently doesn't allow it.

Reply:
nope